### DISCUSSION TITLE
Cubase Midi Remote script for iCON Qcon Pro

### DISCUSSION THREAD

**Discussion by Tichard** [bjoluc/cubase-mcu-midiremote#52] [Show and tell]
So Hi!
First, Thank you for your work, to share the scripts you made for some of the MCU based DAW controllers.
This is a huge help for people trying to get their controller working with the new Cubase MIDI Remote protocol, so flexible, yet so hard to make.

That is why I wanted to share mine. I have a iCON QConPro (original) version, that has a different MIDI layout than the QConPro G2 you made.
So I started from your code and tweaked it to make it work with the QCon Pro "Cubase Mode".
For reference, the controller "MIDI Learning" mode doesn't work as good, some main Cubase control behaviors are lost.

Of course my version is customized for my needs, as I grabbed the opportunity to get rid of the plastic "Cubase/Nuendo" labels template, and removed some of the customizable option you offered in your scripts.
But it would be still easy to make it generic again.

Here is my repository if you are interested to add it to your list of supported device or just curious to see what changed.
https://github.com/Tichard/midiremote-qconpro

I Hope it would help at least someone, as much as this repository helped me <3

**Comment by SebGo79** [comment]
Hi @Tichard 
Would be possible to provide a pre compiled version of your script?
I cant find any download links for it and unfortunatley i am not able to compile all this stuff by myself ;-)

are the master meters working in your script?

UPDATE: Oh...i just saw that you are NOT talking about the Pro X...too bad
So my request makes no sense i think ;-)

best regards
Sebastian

**Reply by Tichard** [reply]
Hi Sebastian
Indeed you got me confused when you asked about the master meters. I wanted to check it before sending the script to you, then realized the QConPro doesn't have a master meter 😆

So I guess you don't need my script.
Feel free to reach out to me if you do in the future.
